### DISCUSSION TITLE
Why are some links to relative Markdown files not working?

### DISCUSSION THREAD

**Discussion by zhumeishuai** [squidfunk/mkdocs-material#7249] [Q&A] [answered]
![image](https://github.com/squidfunk/mkdocs-material/assets/113532348/80202ca0-b3d6-49f6-a95d-447878e3ca8b)
![image](https://github.com/squidfunk/mkdocs-material/assets/113532348/05cd6069-02cd-4944-aeac-a15f85ff2324)
为什么会出现这种情况

**Comment by kamilkrzyskow** [comment]
Hello @zhumeishuai,
this is an English-only discussion board, so please communicate in English. Perhaps @W1ndys could point you to some kind of Chinese help forum if English isn't acceptable for you.

As for what I gathered from basic translations tools, I see you're asking about why the `..\..` path didn't work.
Check your console/terminal output, as it might show a warning with a hint. Maybe you added too many `../` or too little.

Personally I also wouldn't use Windows paths with `\` dashes instead of `/`, which are standard for web browsing.

**Comment by w1ndys** [comment, marked as answer]
# Chinese
@zhumeishuai 
我认为 @kamilkrzyskow 说的没错，在Web站点中应该使用`/`而不是`\`

为此我刚刚测试成功了，请看我以下日志

我的目录层级
```
├─test1
│  ├─test1-1
│  │      test1-1-1.md
│  │      test1-1-2.md
│  │      
│  └─test1-2
│          test1-2.md
│
└─test2
        test2.md
```

我的`test1-1-1.md`
```markdown
# 测试 1-1-1

[test1-1-2.md](test1-1-2.md)

[test1-2.md](../test1-2/test1-2.md)

[test2.md](../../test2/test2.md)
```

其他Markdown文件内容：
```markdown
# 对应标题

test successful
```

总的来说就是，如果被插入的文件和主文件在同一个目录下，不需要任何`/`（例如test1-1-1.md），被插入的文件在主文件所在目录的同级目录下，需要带一层`../`（例如test1-2.md），被插入文件在主文件所在目录的所在目录的同级目录下，则需要加两层`../`（例如test2.md）

以上只是我的一些拙见，如有错误还望指正

![image](https://github.com/squidfunk/mkdocs-material/assets/109416673/844f6ea3-2bc2-449d-8dfa-534d314d621e)

![image](https://github.com/squidfunk/mkdocs-material/assets/109416673/7fd2056d-7860-4925-addc-862e9c4ee021)

---

# English

I think @kamilkrzyskow is right, we should use `/` instead of `\` in the website

I just tested it successfully, please see my log below

My directory hierarchy
```
├─test1
│ ├─test1-1
│ │ test1-1-1.md
│ │ test1-1-2.md
│ │
│ └─test1-2
│ test1-2.md
│
└─test2
test2.md
```

My `test1-1-1.md`
```markdown
# Test 1-1-1

[test1-1-2.md](test1-1-2.md)

[test1-2.md](../test1-2/test1-2.md)

[test2.md](../../test2/test2.md)
```

Other Markdown file contents:
```markdown
# Corresponding title

test successful
```

In general, if the inserted file is in the same directory as the main file, no `/` is needed (e.g. test1-1-1.md). If the inserted file is in the same directory as the main file, it needs to have a layer of `../` (e.g. test1-2.md). If the inserted file is in the same directory as the main file, it needs to have two layers of `../` (e.g. test2.md).

The above are just some of my humble opinions. If there are any mistakes, please correct me.

![image](https://github.com/squidfunk/mkdocs-material/assets/109416673/844f6ea3-2bc2-449d-8dfa-534d314d621e)

![image](https://github.com/squidfunk/mkdocs-material/assets/109416673/757c0c10-0932-43a2-9260-662431130906)

**Comment by squidfunk** [comment]
Please note that it's mandatory to use English on our discussion board. We're responsible for ensuring that users comply with our code of conduct and community standards, and we can't guarantee that in languages other than English. Thank you.

I'm closing and locking this discussion for the reasons stated.
